src/archive/runETL.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Apr  9 22:45:43 2019
+
+@author: zackkingbackup
+"""
+
+from Extract import extract_function
+from Transform import transform_function
+from Load import load_function
+import argparse
+import boto3
+
+''' use argparse.parser to pick off command line args '''
```

Comment:
If you encapsulate this code as a `cli_handler` function, you could put both `lambda_handler` and `cli_handler` in the same file -- `myetl/__init.py__` by preference.